client/src/app/shared/resourceDescriptors.ts

```diff
@@ -198,6 +198,15 @@ export class ArmSiteDescriptor extends ArmResourceDescriptor {
 
     return this._websiteId;
   }
+
+  getTargetAppFormattedName(): string {
+    if (!this._websiteId) {
```

Comment:
what is this? and why are you returning this.site in else

Reply:
fixed